src/script/lua_api/l_client.cpp

```diff
@@ -178,6 +179,35 @@ int ModApiClient::l_disconnect(lua_State *L)
 	return 1;
 }
 
+// unpause()
+int ModApiClient::l_unpause(lua_State *L)
+{
+	g_gamecallback->unpause();
+	//lua_pushboolean(L, true);
+	return 1;
+}
+
+int ModApiClient::l_exit_to_os(lua_State *L)
+{
+	g_gamecallback->exitToOS();
+#ifndef __ANDROID__
+	RenderingEngine::get_raw_device()->closeDevice();
```

Comment:
pretty sure the relevant code already calls this, why is this here?

Reply:
You mean the android snippet?

Reply:
yes

Reply:
it wasnt my code.

I agree its weird. I can snip it out though.